### PR TITLE
[Caffe2ImporterTest] Allow definition of test data path

### PR DESCRIPTION
**Pull request by bertmaher**
*Description*: To make it easier to integrate into build systems with different roots.
*Testing*: ninja Caffe2ImporterTest
*Documentation*: n/a

### CODE REVIEW

**Comment by jfix71** [comment]
Looks like you didn't set `GLOW_DATA_PATH` anywhere?

**Comment by bertmaher** [comment]
Sure I did! ... in the fbcode TARGETS file.  #facepalm

File: tests/unittests/Caffe2ImporterTest.cpp [hunk 1]
```diff
@@ -29,8 +29,10 @@ TEST(caffe2, importConv) {
   auto &mod = EE.getModule();
   Function *F = mod.createFunction("main");
 
-  std::string NetDescFilename("tests/models/caffe2Models/predict_net.pbtxt");
-  std::string NetWeightFilename("tests/models/caffe2Models/init_net.pbtxt");
+  std::string NetDescFilename(GLOW_DATA_PATH
```
Comment (rdzhabarov) [inline tests/unittests/Caffe2ImporterTest.cpp]: Also, similar issue with ONNX importer test.

missing,
#ifndef GLOW_DATA_PATH
#define GLOW_DATA_PATH ""
#endif

**Comment by rdzhabarov** [comment]
Good to go?

https://github.com/pytorch/glow/pull/2329 needs to have that prefix as well once merged.

**Comment by bertmaher** [comment]
Merged!  I'll ping that PR

**Pull request merged**